src/api.ts

```diff
@@ -21,7 +21,7 @@ router.post('/', async (req, res) => {
     const result = await typedData(req);
     return res.json(result);
   } catch (e) {
-    log.warn(`[ingestor] msg validation failed (typed data)`, e);
+    log.warn(`[ingestor] msg validation failed (typed data): ${e}`);
```

Comment:
This wont work if the error is an object

Reply:
checked with
```
const e = new Error("Cannot find module 'express'");
log.warn(`[ingestor] Wrong envelope format ${e}`);
```
and it outputs
```
warn: [ingestor] Wrong envelope format Error: Cannot find module 'express'
```
Can you provide an example of an error we need to handle? BTW, I have changed it because it wasn't work

Reply:
Not sure if this apply in that case, but sometime, the error is a JSON-RPC error object

Reply:
followed the [same approach](https://github.com/snapshot-labs/snapshot-sequencer/pull/146/files#diff-769911c416ccf8514d8fd941ae0abe8fb5c606ade0c218e22151a5f5f9f3d700R47)